rc/extra/pony.kak

```diff
@@ -0,0 +1,113 @@
+# http://ponylang.org
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .*[.](pony) %{
+    set buffer filetype pony
+}
+
+hook global BufOpen .*[.](pony) %{
+    set buffer filetype pony
+	echo -debug "BufOpen"
+}
+
+hook global BufNew .*[.](pony) %{
+    set buffer filetype pony
+}
+
+# Highlighters & Completion
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+addhl -group / regions -default code pony \
+    double_string '"""' '"""'            '' \
+    double_string '"'   (?<!\\)(\\\\)*"  '' \
+    comment       '/\*'   '\*/'            '' \
+    comment       '//'   '$'             ''
+
+addhl -group /pony/double_string fill string
+# addhl -group /pony/single_string fill string
+addhl -group /pony/comment       fill comment
+
+
+%sh{
+    # Grammar
+    values="true|false|None|this"
+    meta='use'
+    # Keyword list is collected using `keyword.kwlist` from `keyword`
+    keywords="and|as|or|break|match|continue|else|try|in|return|end|for|is"
+    keywords="${keywords}|recover|consume|error|do|then|if|while"
+    func_decl="new|fun|be|lambda"
+    capabilities="iso|ref|box|tag|val|trn"
+    type_decl="class|actor|interface|trait|primitive|type"
+    struct="${type_decl}|var|let|embed"
```

Comment:
`struct` and `type_decl` are overlapping here, isn't there a misnaming mistake going on?